recipes/metamorpheus/build.sh

```diff
@@ -3,13 +3,13 @@ set -eox pipefail
 
 PREFIX=$(echo "${PREFIX}" | tr '\\' '/')
 
-mkdir -p $PREFIX/bin
-cp -r $SRC_DIR/* $PREFIX
+mkdir -p $PREFIX/bin/tools
```

Comment:
`tools` is a subfolder of `$PREFIX/dotnet` rather than `$PREFIX/bin`

Reply:
With dotnet 5 `DOTNET_ROOT` defines the folder where `dotnet` is installed (it's different on unix/win)

Reply:
Got it; thanks for the help. Just pushed that change.